client/lib/graph/index.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * External dependencies
+ */
+import { graphql, buildSchema } from 'graphql';
+import createPostResolver from './resolvers/post';
+import createPostsResolver from './resolvers/posts';
+import createPostStatResolver from './resolvers/post-stat';
+import createPostsRequestResolver from './resolvers/posts-request';
+
+function createGraph( store ) {
+	const schema = buildSchema( `
+		type PostDiscussion {
+			comment_count: Int
+		}
+
+		type Post {
+			ID: Int!
+			date: String
+			title: String
+			URL: String
+			like_count: Int
+			discussion: PostDiscussion
+			stat(stat: String): Int
+		}
+
+		input PostQuery {
+			status: String
+			number: Int
+		}
+
+		type Requests {
+			posts(siteId: Int, query: PostQuery): Boolean
```

Comment:
We might want to start thinking about spacing guidelines for schemas and queries. While this is more conventional in GraphQL, our standards typically have spaces within parentheses.